### PR TITLE
refactor: load .env files once, from paths

### PR DESCRIPTION
**Pull request by epoyraz**
`helpers.py`, `daemon.py` and `admin.py` each carried an identical private `_load_env`/`_load_env_file` pair. Moved once into `paths.load_env()` (repo `.env`, then workspace `.env`; set variables win — same order and rule as before) and called from the three modules at import. Net −40 lines, no behaviour change.

Tests: `tests/unit/test_paths.py` covers skipped comments/blanks/non-pairs, quote stripping, no override of set variables, missing file ignored. 152 passed.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: src/browser_harness/paths.py [hunk 1]
```diff
@@ -47,3 +47,16 @@ def tmp_dir() -> Path:
...
+    """Seed os.environ from the repo's .env, then the workspace's .env; set variables win."""
+    for p in files or (Path(__file__).resolve().parents[2] / ".env", workspace_dir() / ".env"):
+        if not p.exists():
+            continue
+        for line in p.read_text(encoding="utf-8-sig", errors="replace").splitlines():
+            line = line.strip()
+            if not line or line.startswith("#") or "=" not in line:
+                continue
+            k, v = line.split("=", 1)
+            os.environ.setdefault(k.strip(), v.strip().strip('"').strip("'"))
```
[the remark this answers is not in the document]
Comment (epoyraz) [reply src/browser_harness/paths.py:62]: Intentionally unchanged: this PR moves the existing parser verbatim so it stays a pure refactor with no behaviour change, and the three copies it replaces all had this stripping. A value that merely ends with a quote is an edge case nobody has hit; if the maintainers want the wrapped-quotes-only rule it is a two-line follow-up, kept separate so it can be reviewed as a semantics change.

File: tests/unit/test_paths.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+def test_load_env_seeds_missing_variables_only(tmp_path, monkeypatch):
+    monkeypatch.setenv("BH_TEST_SET", "kept")
+    monkeypatch.delenv("BH_TEST_NEW", raising=False)
+    monkeypatch.delenv("BH_TEST_QUOTED", raising=False)
+    (tmp_path / ".env").write_text(
+        "# comment\n\nBH_TEST_SET=overridden\nBH_TEST_NEW = fresh\nBH_TEST_QUOTED='q'\nnot a pair\n"
+    )
+
+    paths.load_env(tmp_path / ".env", tmp_path / "missing.env")
```
[the remark this answers is not in the document]
Comment (epoyraz) [reply tests/unit/test_paths.py:13]: Fixed in bfb201b: the test now points `os.environ` at a plain dict for its duration and asserts the whole dict, so nothing leaks into the process environment.
